agenta-backend/agenta_backend/models/db_models.py

```diff
@@ -1,303 +1,470 @@
+
+    id = Column(
+        UUID(as_uuid=True),
+        primary_key=True,
+        default=uuid.uuid7,
+        unique=True,
+        nullable=False,
+    )
+    environment_id = Column(UUID(as_uuid=True), ForeignKey("environments.id"))
+    environment = relationship("AppEnvironmentDB")
+    revision = Column(Integer)
+    modified_by_id = Column(UUID(as_uuid=True), ForeignKey("users.id"))
+    modified_by = relationship("UserDB")
+    deployed_app_variant_revision_id = Column(Integer)
+    deployment_id = Column(Integer)
+    created_at = Column(
+        DateTime(timezone=True), default=lambda: datetime.now(timezone.utc)
+    )
+
+
+class TemplateDB(Base):
+    __tablename__ = "templates"
+
+    id = Column(
+        UUID(as_uuid=True),
+        primary_key=True,
+        default=uuid.uuid7,
+        unique=True,
+        nullable=False,
+    )
+    type = Column(Enum(TemplateType), default=TemplateType.IMAGE, nullable=False)
+    template_uri = Column(String)
+    tag_id = Column(Integer)
+    name = Column(String, unique=True)
+    repo_name = Column(String)
+    title = Column(String)
+    description = Column(String)
+    size = Column(Integer)
+    digest = Column(String)  # sha256 hash of image digest
+    last_pushed = Column(
+        DateTime(timezone=True), default=lambda: datetime.now(timezone.utc)
+    )
+
+
+class TestSetDB(Base):
+    __tablename__ = "testsets"
+
+    id = Column(
+        UUID(as_uuid=True),
+        primary_key=True,
+        default=uuid.uuid7,
+        unique=True,
+        nullable=False,
+    )
+    name = Column(String)
+    app_id = Column(UUID(as_uuid=True), ForeignKey("app_db.id"))
+    app = relationship("AppDB")
+    csvdata = Column(JSONB)
+    user_id = Column(UUID(as_uuid=True), ForeignKey("users.id"))
+    user = relationship("UserDB")
+    created_at = Column(
+        DateTime(timezone=True), default=lambda: datetime.now(timezone.utc)
+    )
+    updated_at = Column(
+        DateTime(timezone=True), default=lambda: datetime.now(timezone.utc)
+    )
+
+
+class EvaluatorConfigDB(Base):
+    __tablename__ = "evaluators_configs"
+
+    id = Column(
+        UUID(as_uuid=True),
+        primary_key=True,
+        default=uuid.uuid7,
+        unique=True,
+        nullable=False,
+    )
+    evaluation_id = Column(UUID(as_uuid=True), ForeignKey("evaluations.id"))
+    evaluation = relationship("EvaluationDB", back_populates="evaluator_configs")
+    evaluation_scenario_id = Column(
+        UUID(as_uuid=True), ForeignKey("evaluation_scenarios.id")
+    )
+    evaluation_scenario = relationship(
+        "EvaluationScenarioDB", back_populates="evaluator_configs"
+    )
+    app_id = Column(UUID(as_uuid=True), ForeignKey("app_db.id"))
+    app = relationship("AppDB")
+    user_id = Column(UUID(as_uuid=True), ForeignKey("users.id"))
+    user = relationship("UserDB")
+    name = Column(String)
+    evaluator_key = Column(String)
+    settings_values = Column(JSONB, default=dict)
```

Comment:
Same as [this](https://github.com/Agenta-AI/agenta/pull/1753/files#r1629550014).

Reply:
we don't have a model for this onre